lib/oli/rendering/content.ex

```diff
@@ -121,11 +121,8 @@ defmodule Oli.Rendering.Content do
         %{"type" => "content", "children" => children} = element,
```

Comment:
This fixes the direction at the wrong level of abstraction.

Note: the new markdown renderer that @darrensiegel is working on will need to add a content writer function.